source/tutorials/index.md

```diff
@@ -1,7 +1,10 @@
 # Tutorials
 
+This section contains a collection of articles on how to achieve a goal with the Nix ecosystem.
```

Comment:
```suggestion
This section contains a collection of articles on how to get thing done in the Nix ecosystem.
```

Reply:
s/thing/things

Reply:
Feel free to ninja-edit my typo-ridden suggestions.